package.json

```diff
@@ -32,6 +32,7 @@
     "build:packages": "gulp build",
     "build": "gulp build",
     "postbuild": "gulp build:install-all-cli-dependencies && gulp build:all-clis",
+    "checkTestsCompile": "echo \"Checking that test source compiles\" && tsc --noEmit -p tsconfig-tests.json",
```

Comment:
What do you think about adding it in the postbuild npm script?

Reply:
That's fine with me. I will change it